doc/configuration.md

```diff
@@ -12,6 +12,8 @@ The library has the following options that can be set as the second parameter wh
         parentContainer: 'body'
     },
     cookieName: 'cookies-accepted',
+    autoAcceptCookieConsentAfterRefreshCount: null, // after this number cookieName will be set automatically
```

Comment:
will be accepted automaticaly.
Die comment kun je ook wel verplaatsen naar de readme, daar staan alle configs uitgelegd.
https://github.com/freshheads/cookie-guard/blob/develop/doc/configuration.md